Common Collections/Strings/Updating a String/src/main.rs

```diff
@@ -0,0 +1,26 @@
+fn main() {
+    // Appending a string slice to a String using the push_str method
+    let mut s = String::from("foo");
+    s.push_str("bar");
+
+    // Using a string slice after appending its contents to a String
+    let mut s1 = String::from("foo");
+    let s2 = "bar";
+    s1.push_str(s2);
+    println!("s2 is {}", s2);
+
+    // Adding one character to a String value using push
+    let mut s = String::from("lo");
+    s.push('l');
+
+    // Using the + operator to combine two String values into a new String value
+    let s1 = String::from("Hello, ");
+    let s2 = String::from("world!");
+    let s3 = s1 + &s2; // note s1 has been moved here and can no longer be used
```

Comment:
Here and in several other places after running the code user gets a warning:
```
warning: unused variable: `s3`
  --> Common Collections\Strings\Updating a String\src\main.rs:19:9
   |
19 |     let s3 = s1 + &s2; // note s1 has been moved here and can no longer be used
   |         ^^ help: if this is intentional, prefix it with an underscore: `_s3`
   |
   = note: `#[warn(unused_variables)]` on by default
```
We might add the prefixes in question for the code to seem "cleaner", but it is up for discussion.

Reply:
I'm still looking for a better solution. I hate these warnings, they are too noisy and absolutely useless. Adding `_` everywhere or disabling warnings in every file seems quite annoying too. I'll come up with something later.